sys_amd64.s

```diff
@@ -154,8 +154,7 @@ TEXT callbackasm1(SB), NOSPLIT|NOFRAME, $0
 
 	POP_REGS_HOST_TO_ABI0()
 
-	MOVQ  0(SP), R10 // get the SP back
-	ADDQ  $8, SP
+    POPQ R10 // get the SP back
```

Comment:
tab?

Reply:
yeah I fixed it. Sorry about making another change. I realized I could save two instructions by actually pushing and popping